base/doc/ltnews40.tex

```diff
@@ -229,6 +229,24 @@ \section{Handling paragraph continuation}
 
 \section{New or improved commands}
 
+\subsection{Avoid bogus \enquote{no item} error}
+
+The commands \cs{addvspace} and \cs{addpenalty} generated the famous
+error message \enquote{Something's wrong--perhaps a missing \cs{item}}
+when they were encountered outside vertical mode. Most of the time this
+error was bogus and if not, then it was generated several times rather
+than once.
+
+Once upon a time (in \LaTeX{}~2.09) it was necessary that these
+commands were used only in vertical mode, but with \LaTeX2e in 1994,
+we changed the internals but simply overlooked that this error became
+useless already then. In this release, i.e. 30 years too late we have
```

Comment:
```suggestion
commands were used only in vertical mode, but with \LaTeXe{} in 1994,
we changed the internals but simply overlooked that this error which
became useless then. In this release, i.e.,~30 years too late, we have
```

Reply:
Suggestion doesn't quite auto-replace the right lines, but you get the idea!

Reply:
took me a whilem but just changes to one line I guess. bit weird the display. wonder what would happen if I would press "commit suggestion"

Reply:
Idea is to change lines 241-243, but somehow I've only managed to get GitHub to link this to line 243 :(

Reply:
then obviously I didn't get it. please make the suggested changes directly on the branch file.

Reply:
I have - next time I'll get it right first time ;)